RealmTasks Apple/Podfile

```diff
@@ -3,7 +3,8 @@ source 'https://github.com/CocoaPods/Specs.git'
 abstract_target 'RealmTasks' do
     use_frameworks!
     
-    pod 'RealmSwift', '~> 2.0.0'
+    pod 'Realm', git: 'https://github.com/realm/realm-cocoa.git', branch: 'tg/changeset-skip', submodules: true
```

Comment:
Until it's merged and released, how else do you expect we can build with those changes?

These Podfile changes be reverted once it's released.